howto/install/deploy-bare-metal.md

```diff
@@ -0,0 +1,93 @@
+To deploy Anbox Cloud on a public cloud (such as AWS, Azure or Google) or using MAAS or OpenStack, see the instructions in [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/deploy-anbox-cloud-with-juju/17744).
+
+Alternatively, you can follow the instructions in this document to use the [manual cloud provider](https://jaas.ai/docs/manual-cloud) that Juju offers. This method allows you to deploy Anbox Cloud with Juju on a set of SSH connected machines.
+
+## Prerequisites
+
+Before you start the installation, ensure that you have the required prerequisites:
+
+* At least three Ubuntu 18.04 LTS or 20.04 LTS machines. See [Minimum hardware](https://discourse.ubuntu.com/t/requirements/17734#minimum-hardware) for details and recommendations.
+* Your *Ubuntu Advantage for **Applications*** token. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a UA Applications token, sign in on https://ubuntu.com/advantage to retrieve it.
+  [note type="caution" status="Warning"]The *Ubuntu Advantage for **Infrastructure*** token that every user gets for free for personal use does **NOT** work and will result in a failed deployment. You must purchase a *Ubuntu Advantage for **Applications*** subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+
+## Install Juju on one machine
```

Comment:
Do we need Juju on one machine (and if so, which one?) or on all machines?

Reply:
Maybe we should make a distinction between the Juju client and the Juju controller
The client (which is installed by running the `snap install` command below) is just a client. It can be connected to a controller running anywhere. You can install as many clients as you want, it's just an entrypoint to the controller
The controller on the other hand, should be installed on one machine (or multiple if you run in HA, but that's a different topic)

Here specifically, the juju client should be installed on the developer machine, not one of the machines mentioned in the bundle

Reply:
Correct !

Juju only needs SSH access to create its controller, nothing else